### DISCUSSION TITLE
Filesystem (local disk) MA in Docker

### DISCUSSION THREAD

**Discussion by robertjb66** [music-assistant/support#3044] [Q&A] [answered]
I run MA in Docker.
Now I want MA to also use the music files I have on my local system. This is the system Docker is running on.
What provider do I select, and how do I point to the /media/robert/Media2/IiTunes folder on my system?
This is not the /media folder that is shown in MA as I believe the /Media in MA is in Docker.

**Comment by OzGav** [comment]
By the sound of it would have to share it and use the remote share provider.

**Comment by robertjb66** [comment]
Thanks thought of that as well. What would be the right info to enter?
- For server, I enter the Ip address of the machine itself.
- For share I enter /media 
- For Subfoler I enter /robert/Media 2/iTunes

On safe I get an error Invalid 192.168.2.3:8095 Invalid share name

**Reply by OzGav** [reply, marked as answer]
Correct the share name will not have a /
